mobile/tests/test_firebase.sh

```
@@ -675,287 +505,47 @@ test_review_list() {
-# Edge Cases & Integration
+# Edge Cases
 # =========================
-test_array_deduplication() {
-  log_test "Edge: Array union prevents duplicates"
+test_array_operations() {
+  log_test "Edge: Array union/removal"
   local id="${EVENT_IDS[0]}"
   
-  # Add same event to user twice
+  # Add same event twice (should deduplicate)
   for i in 1 2; do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)